cypress/app_commands/clean.rb

```diff
@@ -6,7 +6,7 @@
   DatabaseCleaner.clean
 else
   # Manual cleaning
-  true
+  Collection.update_all(druid: nil)
```

Comment:
Can we remove the clause above? We don't have DatabaseCleaner. I'm also curious why you are only updating the druid and not doing `Collection.destroy_all`?

Reply:
`Collection.destroy_all` requires the collection head be set to nil first. This is the least bit of clean-up to allow collections to be created with identical druids. I've added a comment to better explain.